manifests/k/KDE/Dolphin/Nightly/master-b6ec4b1c9/KDE.Dolphin.Nightly.installer.yaml

```
@@ -2,15 +2,15 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
 
 PackageIdentifier: KDE.Dolphin.Nightly
-PackageVersion: master-b6ec4b1c9
+PackageVersion: master-271

```

```suggestion
PackageVersion: master-b6ec4b1c9
```
`PackageVersion` must match Control Panel.